medichaser.py

```diff
@@ -57,6 +57,7 @@
 
 from notifications import (
     gotify_notify,
+    prowl_notify
```

Comment:
you missed comma `,` here and it result in syntax error during tests

Sorry for no guideline included for contribution, but you can try to run "pytest" locally, or you may try to run in in your fork